bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerHandler.java

```diff
@@ -108,6 +111,13 @@ private State getValue(String channelId, GoEStatusResponseDTO goeResponse) {
                     return UnDefType.UNDEF;
                 }
                 return new QuantityType<>(goeResponse.maxCurrent, Units.AMPERE);
+            case MAX_CURR_TEMP:
+                if (goeResponse.maxCurrTemp == null) {
+                    return UnDefType.UNDEF;
+                }
+                logger.debug("MAX_CURR_TEMP, amx ?", goeResponse.maxCurrTemp);
```

Comment:
This seems for debugging purposes only. Please remove the logging. See https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
I will delete the debugging lines.